### ISSUE TITLE
Penumbra and Umbra won't display on imac

### ISSUE THREAD

**Issue by MADAMastronomer** [Stellarium/stellarium#3175]
The Penumbra and Umbra wont display when we use the iMac in the Adelaide Planetarium. The Imac is used for the fulldome.
Tried everything. Please advise.

**Comment by alex-w** [comment]
Screenshot from the MacBook Pro

![stellarium-001](https://user-images.githubusercontent.com/88731/231665618-4549e9e5-e959-4bc3-aeb1-83fc9801688d.png)

Please describe steps to reproduce the issue.

**Comment by gzotti** [comment]
When the Moon is scaled up, the shadow circles are suppressed because they make no sense.

**Comment by alex-w** [comment]
@gzotti should we add note to the tooltip like "Will not show when the Moon scaling is enabled"?

**Comment by MADAMastronomer** [comment]
Yes, a great idea.  I think it would help others, especially when demonstrating a Lunar Eclipse.

Kind Regards,

Mary Adam
Astronomer/Astronomy Educator
Adelaide Planetarium
University Of South Australia
Mawson Lakes
GPO Box 2471, Adelaide SA 5001<x-apple-data-detectors://1/0>
PH: +61 411 519 468<tel:+61%20411%20519%20468>
         08 8302 3138<tel:08%C2%A08302%C2%A03138>
email: ***@***.******@***.***>
web: www.unisa.edu.au/planetarium/<http://www.unisa.edu.au/planetarium/><http://www.unisa.edu.au/planetarium/>





UniSA acknowledges that we respect the Kaurna, Barngarla and Boandik peoples spiritual relationship with their country.  We recognise and respect their cultural heritage, beliefs and relationship with the land. We acknowledge that they are of continuing importance to those people living today. We acknowledge the diversity of Aboriginal peoples, past and present.

On 13 Apr 2023, at 10:35 pm, Alexander V. Wolf ***@***.***> wrote:

﻿

@gzotti<https://github.com/gzotti> should we add note to the tooltip like "Will not show when the Moon scaling is enabled"?
